server/command.go

```diff
@@ -44,14 +44,13 @@ func getCommand() *model.Command {
 	}
 }
 
-func (p *Plugin) getCommandResponse(responseType, text string) *model.CommandResponse {
-	return &model.CommandResponse{
-		ResponseType: responseType,
-		Text:         text,
-		Username:     GITHUB_USERNAME,
-		IconURL:      p.getConfiguration().ProfileImageURL,
-		Type:         model.POST_DEFAULT,
+func (p *Plugin) postCommandResponse(args *model.CommandArgs, text string) {
+	post := &model.Post{
+		UserId:    p.BotUserID,
+		ChannelId: args.ChannelId,
+		Message:   text,
 	}
+	p.API.SendEphemeralPost(args.UserId, post)
```

Comment:
Mind explicitly ignoring the return value? 2/5
```suggestion
	_ = p.API.SendEphemeralPost(args.UserId, post)
```